### PR TITLE
combined interactive component with scene setup

### PR DESCRIPTION
**Pull request by leighembee**

### CODE REVIEW

File: client/components/Floor.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import 'aframe'
 import { Entity } from 'aframe-react'
 import 'babel-polyfill'
 
-const Cursor = () => {
+const Floor= () => {
```
Comment (schai6) [inline client/components/Floor.js:6]: Thanks!

File: client/components/FirstVendor.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import 'aframe'
 import { Entity } from 'aframe-react'
 import 'babel-polyfill'
+import FirstVendorStoreFront from '../components/FirstVendorStoreFront'
```
Comment (schai6) [inline client/components/FirstVendor.js]: I suppose you could just do `import { FirstVendorStoreFront } from './index'`

File: client/components/FirstVendorStoreFront.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
```
Comment (schai6) [inline client/components/FirstVendorStoreFront.js:1]: Need to sync our linters, cause there are a lot of linter issues. -unused imports/variables, self-enclosing components, semi-colons, spacing

File: client/src/index.js [hunk 4]
```diff
@@ -17,21 +17,29 @@ const Main = () => {
...
       <FirstVendor />
       <Cursor />
       <Box />
       <Floor />
     </Scene>
   )
-}
 
-ReactDOM.render(<Main />, document.getElementById('main'))
+};
```
Comment (schai6) [inline client/src/index.js]: Remove semicolon, fix spacing underneath

File: client/src/index.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { Component } from 'react'
```
Comment (schai6) [inline client/src/index.js:1]: Do you want to make this a smart component?

**Review by schai6 — COMMENTED** [review]
Nice! Just some formatting/linter stuff.

File: client/components/FirstVendorStoreFront.js (hunk 3 above)
Comment (leighembee) [reply client/components/FirstVendorStoreFront.js:1]: okay! Also i was under the impression we are using semi colons.. maybe i misunderstood, lets sync our linters tomorrow I am fixing your other suggestions right now, thanks!

Comment (schai6) [reply client/components/FirstVendorStoreFront.js:1]: Ohhhh okay, I forgot. Yeah let's work it out tomorrow then.

File: client/src/index.js (hunk 5 above)
Comment (leighembee) [reply client/src/index.js:1]: Yes! Thank you

**Review by schai6 — APPROVED** [review]
Okay merging!

**Pull request merged**